src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+                        String tLabel = response.substring(tLabelFirst, tTimeFirst - 1);
+                        String tTime = response.substring(tTimeFirst + 4);
+                        Task td = new Event(tLabel, tTime);
+                        t.add(td);
+
+                        System.out.println("Got it! I've added this task: \n" + td.toString());
+                        ctr++;
+                        System.out.println("Now you have " + ctr + " tasks in the list.");
+                        System.out.println("_______________________________________________");
+                    } else if (response.contains("delete")) {
+                        String[] str = response.split(" ");
+                        String task = str[0];
+                        int num = Integer.parseInt(str[1]);
+                        Task tsk = t.remove(num - 1);
+
+                        System.out.println("Noted. I've now removed this task: \n" + tsk);
+                        ctr--;
+                        System.out.println("Now you have " + ctr + " tasks in the list.");
+                        System.out.println("_______________________________________________");
+                    } else {
+                        throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-( " +
+                                "Try todo, event, or deadline");
+                    }
+                } catch (DukeException d) {
+                    System.out.println(d.getMessage());
+                }
+            }
+        }catch(IOException e){
+                System.out.println("OOPS!!! Directory not found. " + e.getMessage());
+                System.exit(0);
+        }
+    }
+    public static void writeFile(File f, ArrayList<Task> tasks) throws IOException {
+        FileWriter fw = new FileWriter(f);
+        for(int i = 0; i < tasks.size(); i++){
+            Task t = tasks.get(i);
+            String text = "";
+            if(t.label.equals("T")){
+                text = text.concat("T|");
+                if(t.isDone){
+                    text = text.concat("1|");
+                }else{
+                    text = text.concat("0|");
+                }
+                text = text.concat(t.description + "\n");
+            }
+            if(t.label.equals("D")){
+                text = text.concat("D|");
+                if(t.isDone){
+                    text =  text.concat("1|");
+                }else{
+                    text = text.concat("0|");
+                }
+                text = text.concat(t.description + "|" + ((Deadline) t).time + "\n");
+            }
+            if(t.label.equals("E")){
+                text =  text.concat("E|");
+                if(t.isDone){
+                    text = text.concat("1|");
+                }else{
+                    text = text.concat("0|");
+                }
+                text = text.concat(t.description + "|" + ((Event) t).by + "\n");
+            }
```

Comment:
I think it is better to add a space around every 'if', 'else' and 'for' as this helps the readability of your code.